tests/performance_tests/performance_test_basic.py

```diff
@@ -228,6 +241,8 @@ def waitForEmptyBlocks(self, node, numEmptyToWaitOn):
         return node.getHeadBlockNum()
 
     def launchCluster(self):
+        if self.clusterConfig.oldNodeos:
+            self.clusterConfig.specificExtraNodeosArgs.update({f"{node}" : "--plugin eosio::history_api_plugin --filter-on \"*\"" for node in range(self.clusterConfig.pnodes, self.clusterConfig._totalNodes)})
```

Comment:
Can this be put into the `ClusterConfig` dataclass's `__post_init__` function when the `testClusterConfig` is created based on how `oldNodeos` is set?